model_compression_toolkit/qat/keras/quantizer/quantization_dispatcher_builder.py

```diff
@@ -33,6 +35,43 @@
                                             qi.QuantizationMethod.UNIFORM: STEUniformActivationQuantizer}}
 
 
+# TODO: move the following "get...config" functions to be members of BaseNode
```

Comment:
Not necessarily to BaseNode, but for some dedicated common location.
I wouldn't create this dependency between BaseNode and TrainableQuantizerWeightsConfig/TrainableQuantizerActivationConfig,
just move this exact function to a common "get_trainable_quantizer" file.